### PR TITLE
MCGA-05:Readme&Info

### PR DESCRIPTION
**Pull request by ojedamilton**
- Created Readme 
- Footer Update 
- Readme Update 
- Structure Collaborators

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -8,10 +8,8 @@ import './index.css';
 
 createRoot(document.getElementById('root')).render(
   <Provider store={store}>
-    <React.StrictMode>
-      <BrowserRouter>
-        <Layout />
-      </BrowserRouter>
-    </React.StrictMode>
+    <BrowserRouter>
```
Comment (BareaMartin) [inline src/index.js]: What was the reason behind this config removal?

Comment (ojedamilton) [reply src/index.js]: Becouse  we had the problem that useEffect run two times in dispatch() ...
[  https://flaviocopes.com/react-useeffect-two-times/](url)

Comment (ojedamilton) [reply src/index.js]:
```
 useEffect(() => {
    dispatch(getProducts());
  }, [dispatch]);
```

Comment (ojedamilton) [reply src/index.js]: Another Solutions ?

**Review by BareaMartin — APPROVED** [review]
I'll leave an approve but please go back with the changes of React.Strictmode removal at index before merging

File: src/index.js (hunk 1 above)
Comment (BareaMartin) [reply src/index.js]: i'll need to sit and check the code for a while to find a two times render component, it's prefferable to re-render some component than avoid checking on a lot of things we are not doing manually, like fragments and components lifecycles

**Comment by ojedamilton** [comment]
ok ! we go back with that change.
tks

**Pull request merged**